pkg/controllers/v1alpha1/dataload/implement.go

```diff
@@ -331,10 +300,49 @@ func (r *DataLoadReconcilerImplement) reconcileExecutingDataLoad(ctx cruntime.Re
 				if jobCondition.Type == batchv1.JobFailed {
 					dataloadToUpdate.Status.Phase = common.PhaseFailed
 				} else {
-					dataloadToUpdate.Status.Phase = common.PhaseComplete
+
+					//  Check existence of the targetPath in alluxio && check dataLoad completely by pod's logs.
+					isComplete := true
+					podList, err := kubeclient.GetPodListByLabel(ctx, job.Namespace, job.Spec.Template.Labels)
+					if err != nil {
+						if apierrs.IsNotFound(err) {
+							log.Error(err, "can't find podList ", "targetDataLoad", targetDataload.Name)
+							isComplete = false
+						}
+					}
+					if len(podList.Items) > 0 {
+						pod := podList.Items[0]
+						logstr, err := kubeclient.GetPodLogs(pod.Name, pod.Namespace, 3)
+						if err != nil {
+							log.Error(err, "Get Pod Logs Error", "targetDataLoad", targetDataload.Name)
+							isComplete = false
+						}
+						if strings.Contains(logstr, "failed as path not exist") {
+							isComplete = false
+							/*
```

Comment:
Suggest to use line comments(`//`) because ["line comments are norm in golang"](https://go.dev/doc/effective_go#commentary)